presto-hive/src/main/java/io/prestosql/plugin/hive/security/SqlStandardAccessControl.java

```diff
@@ -250,6 +251,14 @@ public void checkCanCreateView(ConnectorTransactionHandle transaction, Connector
         }
     }
 
+    @Override
+    public void checkCanRenameView(ConnectorTransactionHandle transaction, ConnectorIdentity identity, SchemaTableName viewName, SchemaTableName newViewName)
+    {
+        if (!isTableOwner(transaction, identity, viewName)) {
```

Comment:
I would assume that you need to be both the table owner **and** the database owner, since this is the equivalent of drop and add, but I see that `canRenameTable` has this behavior.  @electrum @findepi thoughts?